src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/PolldaddyWidgetPageObject.java

```diff
@@ -0,0 +1,83 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.widget;
+
+import org.apache.commons.lang.NotImplementedException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+/**
+ * @ownership: Content X-Wing
+ */
+public class PolldaddyWidgetPageObject extends WidgetPageObject {
+
+  @FindBy(css = ".PDS_Poll *")
+  private WebElement polldaddyBody;
+
+  private static final String TAG_NAME = "pollydaddy";
+  private static final String ARTICLE_NAME = "PolldaddyWidget";
+  private static final String TAG =
```

Comment:
Is it possible to add two tags with different IDs on one page? If it is, why not to add such test case?